src/app/components/media/MediaActionsBar.js

```diff
@@ -250,22 +249,12 @@ class MediaActionsBarComponent extends Component {
       <div className={classes.root}>
         { media.archived === CheckArchivedFlags.NONE ?
           <div>
-            <AddProjectMediaToProjectAction
+            <MoveProjectMediaToProjectAction
```

Comment:
Perhaps we can now rename this to simply `MoveProjectMediaAction`

Reply:
Done and renamed to `MoveProjectMediaAction`